### ISSUE TITLE
In web user interface I can't find creator of the event

### ISSUE THREAD

**Issue by javesin** [owncloud/calendar#423]
In the web client (oC) doesn't show creator info. When I add ownCloud calendar into Thunderbird+Lightning I'm able to see all details including creator.
### Steps to reproduce
1. Log into ownCloud calendar (web client)
2. create event (my own or other shared calendar)
3. go to event details
### Expected behaviour

Not only settings but also creator should be displayed
### Actual behaviour

No info about creator
### Server configuration

ownCloud version: (see ownCloud admin page)
ownCloud 9.0.0 (stable)
### Client configuration

firefox: latest stable (45.0)
xubuntu/ubuntu: latest stable (LTS 14.04)

**Comment by georgehrke** [comment]
> When I add ownCloud calendar into Thunderbird+Lightning I'm able to see all details including creator.

Really? All the time or only if the events contains attendees?

**Comment by georgehrke** [comment]
- [ ] show value of organizer property when editing event

**Comment by javesin** [comment]
You are right - in Thunderbird+Lightning I don't see creator when no attendee defined. But I think I should be able to know who's the creator of event in shared calendar also in case no attendee added.

**Comment by georgehrke** [comment]
Hm, Do you want that also for events sharees create in clients other than this calendar app?

Then we should create a ticket in another repo

**Comment by javesin** [comment]
In my opinion it makes sense to store/see creator and don't differentiate which client-app creates the event. So if you approve it then please create appropriate new ticket or let me know I'll do it.

**Comment by georgehrke** [comment]
I'll propose it in core and we'll put this ticket on hold till then.

**Comment by georgehrke** [comment]
Issue in core: https://github.com/owncloud/core/issues/24012

**Comment by darknight7** [comment]
I've been solicited by users to find a solution as it is very important that some events show their creator, for organization, accounting, etc.
Is this property even stored in the database? I coulnd't find a way to trace back an event to its creator looking at the tables, but I may be wrong as owncloud may store its data in a way that is not immediately recognizable. Is it possible to retreive it with a hack in the code?
